<launch>
  <node name="processing_node" pkg="lesson_perception" type="perception_node" output="screen">
    <rosparam>
      cloud_topic: "sensor/points"
      world_frame: "world_frame"
      camera_frame: "sensor_base"
      voxel_leaf_size: 0.01 <!-- mm -->
      x_filter_min: -2.5 <!-- m -->
      x_filter_max: 2.5 <!-- m -->
      y_filter_min: -2.5 <!-- m -->
      y_filter_max: 2.5 <!-- m -->
      z_filter_min: -2.5 <!-- m -->
      z_filter_max: 2.5 <!-- m -->
      plane_max_iterations: 100
      plane_distance_threshold: 0.03
      cluster_tolerance: 0.01
      cluster_min_size: 250
      cluster_max_size: 500000
    </rosparam>
  </node>
</launch>
